cmdk/src/index.tsx

```diff
@@ -178,11 +178,7 @@ const Command = React.forwardRef<HTMLDivElement, CommandProps>((props, forwarded
           // Filter synchronously before emitting back to children
           filterItems()
           sort()
-          schedule(1, () => {
-            // Select the first item and emit again
-            selectFirstItem()
-            store.emit()
```

Comment:
we don't need to emit again now because `selectFirstItem` calls into `setState` with the new value and that causes a new emit already